scripts/docker/Dockerfile.clang

```diff
@@ -26,19 +28,28 @@ RUN CMAKE_KEY=2D2CEF1034921684 && \
     rm cmake*
 ENV PATH=${CMAKE_DIR}/bin:$PATH
 
+ARG NPROC=8
+
+# Clone Kokkos fork of the LLVM Project and build Clang
 ENV LLVM_DIR=/opt/llvm
-RUN LLVM_VERSION=8.0.0 && \
```

Comment:
Actually I forgot we were using this image in the clang-format build as well
https://github.com/kokkos/kokkos/blob/90857495562c55735960ac0084f4c29fd7c67321/.jenkins#L246-L249